app/controllers/courses_controller.rb

```diff
@@ -97,6 +107,7 @@ def create
         format.html { redirect_to @course, notice: 'Course was successfully created.' }
         format.json { render :show, status: :created, location: @course }
       else
+        @tags = Tag.all.where.not(course_tags_count: 0)
         format.html { render :new }
```

Comment:
I would put the `@tags=` line inside the format.html block.

Reply:
fixed